scripts/benchmark_cortex.py

```diff
@@ -37,41 +39,34 @@
 NUM_THREADS = 4
 NUM_FORMS = 20
 
-
 def _addRows(rows, core, one_at_a_time=False):
     if one_at_a_time:
         for row in rows:
             core.addRows([row])
     else:
         core.addRows(rows)
-    # core.flush()
-
+        # core.flush()
```

Comment:
This would have come from when I reformatted this file with pycharm.  We likely won't maintain this script over the long-term (note the use of pandas).